src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -383,6 +386,52 @@ public void testAddQuestionAction() throws Exception {
 
         assertEquals("Blank options should have been removed", 2, feedbackEditPage.getNumOfOptionsInRankOptions(1));
 
+        ______TS("Rank edit: Invalid empty input in number of options a respondent must rank");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("RANK_OPTIONS");
+        feedbackEditPage.waitForPageToScroll();
+
+        feedbackEditPage.fillQuestionTextBoxForNewQuestion("filled qn");
+        feedbackEditPage.fillRankOptionForNewQuestion(0, "Option 1");
+        feedbackEditPage.fillRankOptionForNewQuestion(1, "Option 2");
+
+        feedbackEditPage.clickEnableMinRankOptions(NEW_QUESTION_INDEX);
+        feedbackEditPage.clearMinRankOptions(NEW_QUESTION_INDEX);
+
+        feedbackEditPage.clickAddQuestionButton();
+
+        WebElement rankMinOptionInput = browser.driver.findElement(By.id("minOptionsToBeRanked-" + NEW_QUESTION_INDEX));
+
+        assertFalse(feedbackEditPage.isInputElementValid(rankMinOptionInput));
+
+        ______TS("Rank edit: Invalid letters in number of options a respondent must rank");
+        feedbackEditPage.fillMinOptionsToBeRanked(NEW_QUESTION_INDEX, "invalid letters");
```

Comment:
If the input element is already invalid, how do we know that this will cause the input element to be invalid?

Reply:
I think you missed this

Reply:
I think you missed this.